### PR TITLE
Add Pisces Hotspot

### PR DESCRIPTION
**Pull request by piscesminer**
This is to add Pisces to the hotspot app. One thing needs changing is the onboarding ID in src/makers/pisces/index.ts .

### CODE REVIEW

**Review by cokes518 — COMMENTED** [review]
looks good content-wise. @tyler-whitman ready for you to check as well.

File: src/makers/pisces/antennas.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { MakerAntenna } from '../antennaMakerTypes'
+
+const BOBCAT = {
```
Comment (tyler-whitman) [inline src/makers/pisces/antennas.ts:3]: You need to change this to `PISCES` instead of `BOBCAT`

File: src/makers/pisces/index.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 1616, // TODO: Waiting for maker id from onboarding server
```
Comment (tyler-whitman) [inline src/makers/pisces/index.ts:7]: Update maker id once it is approved.

File: src/makers/pisces/pisces.svg [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<svg width="93" height="94" viewBox="0 0 93 94" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (tyler-whitman) [inline src/makers/pisces/pisces.svg:1]: This image looks exactly like bobcat without the text in the middle, is that intended?

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

File: src/makers/pisces/pisces.svg (hunk 3 above)
Comment (piscesminer) [reply src/makers/pisces/pisces.svg:1]: Sorry I submitted the wrong commit.

Comment (piscesminer) [reply src/makers/pisces/pisces.svg:1]: This is the branch before adjustment, I apologize for my mistake

**Comment by piscesminer** [comment]
We are very sorry because we submitted the wrong commit in a hurry, please merge the new commit again. Thanks, and sorry

**Comment by cokes518** [comment]
@piscesminer can you tell me the name you want in the onboarding server and the number of asserts you'd like to pay for your customers? (minimum is 1).

A list of existing maker names: https://onboarding.dewi.org/api/v2/makers

**Comment by piscesminer** [comment]
> @piscesminer can you tell me the name you want in the onboarding server and the number of asserts you'd like to pay for your customers? (minimum is 1).
> 
> A list of existing maker names: https://onboarding.dewi.org/api/v2/makers

Thanks.
{
"name": "Pisces Miner",
"locationNonceLimit": 1,
}
We think so.
The name is "Pisces Miner"
And using the minimum.  👍

**Review by cokes518 — APPROVED** [review]

File: src/makers/pisces/index.ts (hunk 2 above)
Comment (cokes518) [inline src/makers/pisces/index.ts:7]:
```suggestion
  id: 13,
```

**Review by cokes518 — APPROVED** [review]
maker account created: 
```{
    "code": 200,
    "data": {
        "id": 13,
        "name": "Pisces Miner",
        "address": "134C7Hn3vhfBLQZex4PVwtxQ2uPJH97h9YD2bhzy1W2XhMJyY6d",
        "locationNonceLimit": 1
    },
    "success": true
}```

**Comment by cokes518** [comment]
FYI, we need you to update the maker ID, we can't commit the change.

**Comment by carzygod** [comment]
> FYI, we need you to update the maker ID, we can't commit the change.

Hi,we update the maker ID. It is okay?

**Comment by piscesminer** [comment]
> FYI, we need you to update the maker ID, we can't commit the change.

Hi,we update out maker ID . It's okay?

File: src/makers/pisces/hotspots.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        '将电源适开启，并将天线插入 Hotspot 背面并旋紧。',
+        '当指示灯由白变为绿色，即表示 Hotspot 准备就绪。',
+      ],
+      bluetooth: [
+        '使用随附的大头针按下 Hotspot 背面的 BT 按钮，长按 5 秒钟。',
+        '当指示灯由白变为绿色，即表示 Hotspot 准备进行配对。\n\n请务必打开您手机的蓝牙功能！',
+      ],
+    },
+  },
+  icon: HotspotIcon,
```
Comment (tyler-whitman) [inline src/makers/pisces/hotspots.ts:59]: icon property is duplicated, this one needs to be removed. Please run `yarn lint` to see errors and fix them. Some may be able to be fixed with `yarn lint --fix`

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

File: src/makers/pisces/hotspots.ts [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  antenna: { default: ANTENNAS.pisces },
```
Comment (tyler-whitman) [inline src/makers/pisces/hotspots.ts:60]: This should be `ANTENNAS.Pisces`

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

**Review by piscesminer — COMMENTED** [review]
icon fix

File: src/makers/pisces/hotspots.ts (hunk 5 above)
Comment (piscesminer) [inline src/makers/pisces/hotspots.ts]: okay,and Thanks :+1:

Comment (piscesminer) [reply src/makers/pisces/hotspots.ts]: And really sorry about this mistake....

File: src/makers/pisces/hotspots.ts (hunk 4 above)
Comment (tyler-whitman) [reply src/makers/pisces/hotspots.ts:59]: @piscesminer you still need to remove this line

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

**Review by piscesminer — COMMENTED** [review]
Oh yes, icon property is duplicated. 👍

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**